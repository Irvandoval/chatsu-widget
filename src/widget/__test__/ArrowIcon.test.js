import React from 'react';
import ArrowIcon from '../ArrowIcon';
import { JSDOM } from 'jsdom';
import { shallow } from 'enzyme';

describe('<ArrowIcon />', () => {
  it('matches the snapshot', () => {
    const tree = shallow(<ArrowIcon />);
    expect(tree).toMatchSnapshot();
  });
})