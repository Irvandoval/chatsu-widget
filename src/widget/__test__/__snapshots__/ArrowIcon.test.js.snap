// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ArrowIcon /> matches the snapshot 1`] = `
<div>
  <svg
    fill="#FFFFFF"
    height="15"
    style={
      Object {
        "marginRight": 15,
        "marginTop": 6,
        "verticalAlign": "middle",
      }
    }
    viewBox="0 0 15 15"
    width="15"
    xmlns="http://www.w3.org/2000/svg"
  >
    <line
      stroke="white"
      stroke-width="1"
      x1="1"
      x2="15"
      y1="15"
      y2="1"
    />
    <line
      stroke="white"
      stroke-width="1"
      x1="1"
      x2="15"
      y1="1"
      y2="15"
    />
  </svg>
</div>
`;
